### DISCUSSION TITLE
Where do I download 2.3.1 Win64?

### DISCUSSION THREAD

**Discussion by eisenbruch** [lllyasviel/Fooocus#2712] [General]
In the installation instructions it links to 2.1.831.

The most recent release is 2.3.1 it looks like, but when I download that zip there is no run.bat. I need the win64 version of 2.3.1 I think but can't find links for it.

Thanks in advance for your hard work and help!

**Comment by ILikeToasters** [comment]
Download the one listed for the installation instructions on the main page which is the older version. It will update when you run the run.bat file automatically. That is the recommended way of doing it.

**Reply by eisenbruch** [reply]
Ah, got it! Thank you! That was not clear to me (im dumb)

**Reply by ILikeToasters** [reply]
Not to worry. You are not the first to ask.

**Comment by eisenbruch** [comment]
This can be removed if it is unnecessary :)

**Reply by ILikeToasters** [reply]
I think you can just mark as answered. That way anyone else with same question might see it.
